test/acceptance/kafo_configure_test.rb

```diff
@@ -131,7 +130,7 @@ module Kafo
       end
 
       it 'must create file' do
-        skip 'Requires Puppet 4.5+ for data in modules' if Puppet::PUPPETVERSION < '4.5.0'
+        skip 'Requires Puppet 4.5+ for data in modules' if Gem::Specification.find_all_by_name('puppet').sort_by(&:version).last.version < Gem::Version.new('4.5.0')
```

Comment:
I used the same comparison method as other tests just for consistency, which is to check the gem version rather than require the library and check its version. If you wish, I can refactor them all to use `puppet/version`.